lib/puppet/file_serving/configuration.rb

```diff
@@ -81,6 +82,8 @@ def mk_default_mounts
     @mounts["modules"].allow('*') if @mounts["modules"].empty?
     @mounts["plugins"] ||= Mount::Plugins.new("plugins")
     @mounts["plugins"].allow('*') if @mounts["plugins"].empty?
+    @mounts["locales"] ||= Mount::Locales.new("locales")
+    @mounts["locales"].allow('*') if @mounts["locales"].empty?
```

Comment:
@justinstoller this is the bit I was talking about that seems to set `allow` permissions of some kind for the built-in mountpoints. What was the other change you thought we might need?

Reply:
We shouldn't need any other changes currently.

The thing I mumbled in standup was me working through this path pseudo-in-my-head:

This change is needed for Puppet's internal Ruby based [auth system](https://github.com/puppetlabs/puppet/blob/ebd96213cab43bb2a8071b7ac0206c3ed0be8e58/lib/puppet/network/authstore.rb#L41) (that takes inputs from [fileserver.conf](https://docs.puppet.com/puppet/4.5/config_file_fileserver.html#format) and [legacy auth.conf](https://docs.puppet.com/puppet/4.5/config_file_auth.html#important-this-is-a-deprecated-config-file)). Puppet server manages this kind of authorization through its newer [tk-auth based auth.conf](https://docs.puppet.com/puppetserver/2.4/config_file_auth.html). Both need to allow access for the request to go through, but the default setting for Puppet Server is to [allow access](https://tickets.puppetlabs.com/browse/SERVER-1808) to the `file_*` endpoints so no further work should be needed. However, there's a desire to follow our own best practices and [only allow specific mountpoints](https://github.com/puppetlabs/puppetlabs-puppet_enterprise/pull/1342#issuecomment-340533534) within tk-auth, that work, when/if it is adopted will need to take this into account.